### ISSUE TITLE
bandage

### ISSUE THREAD

**Issue by yolanda-huang** [ubccr/software-layer#163]
http://rrwick.github.io/Bandage/

**Comment by tonykew** [comment]
Test build complete - GUI launches AOK in my tests - more info to follow

**Comment by tonykew** [comment]
Bandage version 0.9.0 has been published to the ccrsoft/2023.01 software release

Note that this is a GUI app so run from CCR OnDemand

    https://ondemand.ccr.buffalo.edu

e.g.
[UB-HPC Cluster Desktop]
Partition [General Compute]
Quality of Service [General Compute]
Number of Nodes "1"
[...]
Node Features "NOTLEGACY"

```
$ module spider bandage
----------------------------------------------------------------------------
  bandage: bandage/0.9.0
----------------------------------------------------------------------------
    Description:
      Bandage is a program for visualising de novo assembly graphs


    You will need to load all module(s) on any one of the lines below before the "bandage/0.9.0" module is available to load.

      gcccore/11.2.0
[...]
$ module load gcccore/11.2.0 bandage/0.9.0
$ Bandage

```
